bundles/mapping/userstyle/resources/locale/fi.js

```diff
@@ -6,11 +6,33 @@ Oskari.registerLocalization(
         "title": "Omat tyylit",
         "styles" : "Tyylit",
         "addStyle" : "Lisää uusi tyyli",
-        "defaultName": "Oma tyyli",        
+        "defaultName": "Oma tyyli",
+        "runtime": "Kirjautumattomana oma tyyli on käytettävissä vain tämän session ajan. Kirjaudu sisään tallentaaksesi tyylin tiedot.",
         "popup": {
             "title": "Oma tyyli",
             "name" : "Tyylin nimi",
             "style" : "Esitystapa"
+        },
+        "tab": {
+            "title": "Omat tyylit",
+            "grid": {
+                "layer": "Karttataso",
+                "name": "Nimi",
+                "created": "Luontiaika",
+                "updated": "Muokkausaika",
+                "actions": "Toiminnot"
+            }
+        },
+        "success": {
+            "post": "Oma tyyli tallennettu",
+            "put": "Oma tyyli päivitetty",
+            "delete": "Oma tyyli poisttettu"
+        },
+        "error": {
+            "get": "Omien tyylien haku epäonnistui",
+            "post": "Oman tyylin tallennus epäonnistui",
+            "put": "Oman tyylin päivitys epäonnisui",
```

Comment:
```suggestion
            "put": "Oman tyylin päivitys epäonnistui",
```
Or maybe we could use the one from post for put as well to reduce translations?